src/Validation/src/Validation.csproj

```diff
@@ -4,7 +4,15 @@
     <OutputType>Exe</OutputType>
     <TargetFrameworks>$(NetCurrent)</TargetFrameworks>
     <IsPackable>true</IsPackable>
-    <SignAssembly>true</SignAssembly>
+  </PropertyGroup>
+
+  <!-- Set package ID for official signed builds. Needed to publish both Windows and Linux signed packages to build asset registry. -->
+  <PropertyGroup Condition="'$(DotNetSignType)' == 'real'">
+    <BuildRid Condition="'$(BuildRid)' == '' and '$(MSBuildRuntimeType)' == 'core'">$([System.Runtime.InteropServices.RuntimeInformation]::RuntimeIdentifier)</BuildRid>
```

Comment:
Why is it necessary to set the Target properties here? As opposed to just the package id.

Reply:
I suppose it's not necessary, this is similar to how we do it in the VMR: https://github.com/dotnet/sdk/blob/619b55dd198bbea2d5021ad0148f82097c379e21/src/SourceBuild/content/Directory.Build.props#L76.

I'll adjust this section to just be <PackageId>Validation.$(OS)</PackageId>

Reply:
The VMR is a bit different here because a lot of those parameters do matter for the artifacts produced. For this package, it shouldn't matter.